### DISCUSSION TITLE
Adding support for Czech

### DISCUSSION THREAD

**Discussion by JiriHeger** [simonwep/ocular#54] [Ideas]
I have two things I would like to change.
1. add the currency there - the Czech crown, either as "Kč" or "CZK".
2. Add the language Czech. I will translate it from English to Czech.
Email me what I have to change the two things you wrote to me.
Thank you.
George
jiri.heger@gmail.com

**Comment by simonwep** [comment]
Hey! That's awesome :) Head over to the [contributing guidelines](https://github.com/simonwep/ocular/blob/main/.github/CONTRIBUTING.md), specifically, after you've set it up, the section [adding support for a new language](https://github.com/simonwep/ocular/blob/main/.github/CONTRIBUTING.md#adding-support-for-a-new-language). Don't hesitate to ask questions if you're having trouble setting it up, I'm currently working on simplifying the "getting-started"-process 😄

**Reply by JiriHeger** [reply]
Thanks for the info, I'll look into it.

**Reply by JiriHeger** [reply]
Ahoj Simone,
[cz.json](https://github.com/user-attachments/files/17562097/cz.json)

I'm not a programmer, so I proceeded as follows for the "cz.json" file:
1. I downloaded the "en.json" file
2. I opened it in VisualStudio Code to see what it looks like.
3. I made a duplicate of the "en.json" file. I called the copy "cs.json"
4. I translated the texts that were intended for translation.

For the types.ts file I just added the currency "CZK"

I am sending them to you to review and put into the application.

Thanks in advance.

[cz.json](https://github.com/user-attachments/files/17562097/cz.json)

**Reply by simonwep** [reply]
That is totally fine, thank you very much!
